### PR TITLE
Up-hint and UI cleanup

### PR DESCRIPTION
**Pull request by skull132**
Adds a UI element to all mobs which indicates whether or not there's a ceiling above them.
Example: https://kama.skullnet.me/index.php/s/1ZYelk4T1vNOr3F

Also cleans up the multiple screen .dmis we have and syndicates them all into one subfolder.

### CODE REVIEW

**Comment by skull132** [comment]
Also, I realized, one of your PRs, which makes open-turf not necessarily be a hole, might start messing with my `isopenturf` usage here? @Lohikar

**Comment by Lohikar** [comment]
Copypaste of my comment about this on Discord:
> for all intents and purposes you should be able to use isopenturf() for that, Zmovement and fall mechanics are likely to stay defined there - it's just openspace (Zmimic) that's being moved

**Comment by skull132** [comment]
\> stale

😠 🗡

**Comment by skull132** [comment]
holy shit fuck that screen object rewrite.

File: code/_onclick/hud/screen_objects.dm [hunk 1]
```diff
@@ -223,6 +223,15 @@
 			if(usr.client)
 				usr.client.drop_item()
 
+		if("up hint")
```
Comment (Lohikar) [inline code/_onclick/hud/screen_objects.dm]: Ehh. Would be better to make a subtype of `/obj/screen` and put this behavior there.

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 2]
```diff
@@ -117,8 +117,16 @@
 
 /mob/living/carbon/human/Move()
 	. = ..()
+
+	var/turf/T = loc
+	if (!istype(T))
```
Comment (Lohikar) [inline code/modules/mob/living/carbon/human/human_movement.dm]: `isturf()` is faster than `istype()`.

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 3]
```diff
@@ -117,8 +117,16 @@
 
 /mob/living/carbon/human/Move()
 	. = ..()
+
+	var/turf/T = loc
+	if (!istype(T))
+		return
+
+	if (client)
+		var/turf/B = HasAbove(T.z) ? get_step(T, UP) : null
```
Comment (Lohikar) [inline code/modules/mob/living/carbon/human/human_movement.dm]: Why not just `GetAbove()`?

File: code/_onclick/hud/screen_objects.dm (hunk 1 above)
Comment (skull132) [reply code/_onclick/hud/screen_objects.dm]: I won't be able to refactor screen objects in time for January update. So I'll just rebase my work on that end to another branch and keep this using the old system.

**Pull request merged**